syntax = "proto3";

package turtle;

message zara_data {
    map<string, int64> dict = 1;
    int32 age = 2;
}